ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellDemoController.swift

```diff
@@ -8,7 +8,7 @@ import UIKit
 
 // MARK: TableViewCellDemoController
 
-class TableViewCellDemoController: UITableViewController {
+class TableViewCellDemoController: DemoTableViewController {
```

Comment:
nit: The entry point for **TableViewCell** and **TableViewCellFileAccesorryView** should be moved up to the "Vnext Control" section of the demo app.